integration/travis/run_integration.sh

```diff
@@ -107,20 +107,13 @@ case "$COOK_POOLS" in
     exit 1
 esac
 
-mkdir ${SCHEDULER_DIR}/log
-
 pip install flask
 export DATA_LOCAL_PORT=35847
 export DATA_LOCAL_SERVICE="http://localhost:${DATA_LOCAL_PORT}"
 export DATA_LOCAL_ENDPOINT="${DATA_LOCAL_SERVICE}/retrieve-costs"
+mkdir ${SCHEDULER_DIR}/log
```

Comment:
This particular change no longer exists. Originally, it was needed to ensure that the logging directory existed, otherwise the flask app would fail to start; that affected both the data local and the (now gone) demohooks flask app.

Reply:
So should we get rid of this?

Reply:
This change doesn't exist in the current patch.